doc/utils/sphinx-config/conf.py.in

```diff
@@ -264,8 +263,18 @@ else:
   html_math_renderer = 'mathjax'
 
 # use relative path for mathjax, so it is looked for in the
-# html tree and the manual becomes readable when offline 
+# html tree and the manual becomes readable when offline
 mathjax_path = 'mathjax/es5/tex-mml-chtml.js'
+
```

Comment:
@hammondkd finally figured out a way to have a "real" angstrom symbol in both HTML output with MathJax and PDF output with LaTeX. Using `\mathring{A}` is a good approximation, but always looked a bit strange because of the additional space between the A and the circle. Strangely, when searching the web I found several workarounds for making HTML output work, but nothing that also can be used with LaTeX. Several suggestions do not even through an error, but just print a warning and drop the character from the output.